README.md

```diff
@@ -148,7 +148,10 @@ The Amazon Chime SDK for JavaScript uses WebRTC, the real-time communication API
 
 ## Installation
 
-Make sure you have Node.js version 12 or higher. Node 14 is recommended and supported.
+Make sure you have the following dependency installed:
+- Node.js version 12 or higher. Node 14 is recommended and supported.
+- Typescript version 4.2.3 (Later version may cause build errors in `lib.d.ts`).
```

Comment:
Does this mean that if you use our SDK you are forced to use 4.2.3 for your whole application? if using typescript i guess. Is there a reference issue?

Reply:
Yeah. later versions have a lot of breaking changes in lib.d.ts that causes a bunch of reference issues.